### ISSUE TITLE
Map Picker for setting a Location Item though the UI

### ISSUE THREAD

**Issue by proggprod** [openhab/openhab-webui#1730]
I'd love to have a Map Picker for setting a Location Item though the UI.

In my example, I want to use it to create a widget for creating a list of pre-defined locations. I currently make it work by looking up google maps and typing the coordinates.
![image](https://user-images.githubusercontent.com/81530084/221149926-6c57e61d-0b62-40a1-afa6-a86622e51b19.png)

**Comment by florian-h05** [comment]
Community https://community.openhab.org/t/openhab-4-0-wishlist/142388/423?u=florian-h05.

**Comment by florian-h05** [comment]
Can you please elaborate what exactly you want?
A map picker component that can be integrated into UI widgets or what?

**Comment by proggprod** [comment]
Yes. I want to be able to click a button, have a map picker popup, then when I select a point, send the location info to the specified Location item.
Maybe the yaml would look something like this:

```
component: oh-button
config:
  action: mapPicker
  actionItem: test_Location_Item
```

**Comment by ghys** [comment]
This is interesting, having a "map picker" action, after all we already have a "photo browser" action.

There is also probably a need for `oh-map-picker`, `oh-map-picker-card` widgets and derivatives. Those would be added in the default widgets determinators (is that a word? apparently not).
See https://github.com/openhab/openhab-webui/issues/411
